client/blocks/signup-form/index.jsx

```diff
@@ -677,15 +651,11 @@ class SignupForm extends Component {
 					{ this.props.formFooter || this.formFooter() }
 				</LoggedOutForm>
 
-				{ ! this.isJetpackSocialABTest() &&
-					this.props.isSocialSignupEnabled &&
-					! this.userCreationComplete() && (
-						<SocialSignupForm
-							handleResponse={ this.props.handleSocialResponse }
-							socialService={ this.props.socialService }
-							socialServiceResponse={ this.props.socialServiceResponse }
-						/>
-					) }
+				<SocialSignupForm
```

Comment:
It looks like this removal was a bit overzelous and the `this.props.isSocialSignupEnabled && ! this.userCreationComplete()` conditions are unrelated and should remain.

Reply:
right! good catch!